### ISSUE TITLE
Mocking get_web_driver to return an additional parameter

### ISSUE THREAD

**Issue by jorythompson** [mrooney/mintapi#206]
I am having a devil of a time trying to understand Mock framework and apply a minor change to this api.  I am changing get_web_driver to return an additional string but am not sure how to start changing the tests to support this.

Any help would be most welcome!

**Comment by jorythompson** [comment]
If someone could point me in the right direction, I could update the test and close out my code update...

**Comment by jorythompson** [comment]
bump

**Comment by jorythompson** [comment]
I think I solved the problem!  Please check the additional classes in the tests.py file:
```
class Attribute:
    text = json.dumps({'response': {'42': {'response': accounts_example}}})


class Element:
    @staticmethod
    def get_attribute(test):
        return json.dumps({'token': '123'})


class TestMock:
    @staticmethod
    def find_element_by_name(test):
        return Element()

    @staticmethod
    def request(a, b, **c):
        return Attribute()
```

**Comment by jorythompson** [comment]
Problem solved and in pull request #205.
